hazelcast-ra/hazelcast-jca/src/test/java/com/hazelcast/jca/HazelcastConnectionImplTest.java

```diff
@@ -191,7 +190,7 @@ public void getJobTracker() {
     @Test
     public void getCluster() {
         Cluster cluster = connection.getCluster();
-        assertNotSame(cluster, hz.getCluster());
```

Comment:
There was a `ClusterProxy` instance wrapping cluster service itself. I just removed it and made cluster service an implementation of Cluster interface.